src/main/resources/story/Imdb/imdb-add-watchlist-export.story

```diff
@@ -0,0 +1,21 @@
+Description: Login into an existing user account. Add a movie into a watchlist, sort the watchlist and export it.
+
+Scenario: Login in.
+When I login to IMDB with <email> and <password>
+
+Scenario: Search a movie and add it to watchlist.
+When I enter `Interstellar` in field located `By.xpath(//*[@id="suggestion-search"])`
+When I wait until element located `By.xpath(//*[@data-testid="search-result--const"]//*[text()="Matthew McConaughey, Anne Hathaway"])` appears
+When I click on element located `By.xpath(//*[@data-testid="search-result--const"]//*[text()="Matthew McConaughey, Anne Hathaway"])`
+When I wait until element located `By.xpath(//*[@data-testid="hero-title-block__title"][text()="Interstellar"])` appears
+When I wait until element located `By.xpath(//*[@class="ipc-btn__text"][text()="Add to Watchlist"])` appears
```

Comment:
[@class="ipc-btn__text"] - do you really need to put a reference to this element into the xpath expression? text() should be enough

Reply:
I didn't need the reference to this element. I've changed the xpath to just text().